### PR TITLE
Docs/mouse

### PR DESCRIPTION
**Pull request by AechDub**
This control is fairly simplistic.  Please read and make sure it makes sense to you :)

### CODE REVIEW

File: src/reference/interactive/index.pug [hunk 1]
```diff
@@ -337,6 +337,17 @@ block reference
...
+    h3#mouse Mouse
+
+    p Mouse input allows you to get mouse events over video.  The mouse input control is pretty simplistic.  You can subscribe to receive mouse move on:
+
+    ul
+        li #[code mousedown] (default) - this will only send mouse move events during a mousedown event
+        li #[code always] mouse move events are sent on hover and during mousedown
+        li #[code never] never send mouse move events.  You will still receive click events.
+
+    p You can also adjust mouse move event throttling.  The #[code moveThrottle] will prohibit move events from sending but only every provided millisecond.  It is suggested to have some limit here, but 0 (no throttling) is allowed.  The default is 50ms.
```
Comment (connor4312) [inline src/reference/interactive/index.pug]: > The #[code moveThrottle] will prohibit move events from sending but only every provided millisecond

I'd reword as

> The #[code moveThrottle] duration, given in milliseconds, will throttle movement events sent to the game client.

Comment (AechDub) [reply src/reference/interactive/index.pug]: Agreed.  Better wording :)
